Mathlib/Tactic/FunProp/AEMeasurable.lean

```diff
@@ -4,64 +4,15 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Tomáš Skřivan
 -/
 import Mathlib.MeasureTheory.Measure.AEMeasurable
-import Mathlib.MeasureTheory.Constructions.Prod.Basic
-
-import Mathlib.Tactic.FunProp.Measurable
 
 /-!
 ## `fun_prop` minimal setup for AEMeasurable
 -/
 
-
-section missing
-
-open MeasureTheory
-
-variable {ι α β γ δ R : Type*} {m0 : MeasurableSpace α} [MeasurableSpace β] [MeasurableSpace γ]
-  [MeasurableSpace δ] {f g : α → β} {μ ν : Measure α}
-
-theorem AEMeasurable.comp_aemeasurable' {f : α → δ} {g : δ → β} (hg : AEMeasurable g (μ.map f))
```

Comment:
Moved this one to `Mathlib/MeasureTheory/Measure/AEMeasurable.lean`. Happy to split that into a separate PR, if preferred.